Kickstarter-iOS/AppDelegate.swift

```diff
@@ -422,6 +426,47 @@ internal final class AppDelegate: UIResponder, UIApplicationDelegate {
     let task = session.dataTask(with: url)
     task.resume()
   }
+
+  private func configureRemoteConfig() {
+    let remoteConfigClient = RemoteConfigClient(with: RemoteConfig.remoteConfig())
+
+    AppEnvironment.updateRemoteConfigClient(remoteConfigClient)
+
+    let appDefaults: [String: Any?] = [
+      RemoteConfigFeature.consentManagementDialogEnabled.rawValue: false,
+      RemoteConfigFeature.facebookLoginInterstitialEnabled.rawValue: false
+    ]
+
+    AppEnvironment.current.remoteConfigClient?.setDefaults(appDefaults as? [String: NSObject])
+
+    AppEnvironment.current.remoteConfigClient?.activate { _, error in
+      guard let remoteConfigActivationError = error else {
+        print("🔮 Remote Config SDK Successfully Activated")
+
+        // TODO: Rename/rework self.viewModel.inputs.didUpdateOptimizelyClient(optimizelyClient)
+        return
+      }
+
+      print("🔴 Remote Config SDK Activation Failed with Error: \(remoteConfigActivationError.localizedDescription)")
+
+      Crashlytics.crashlytics().record(error: remoteConfigActivationError)
+
+      self.viewModel.inputs.optimizelyClientConfigurationFailed()
+      // TODO: Rename/rework input signal with Optimizely gone.
```

Comment:
We'll discuss how to change these in a 1-1 Monday (`line 446` and `line 455`)
Should be pretty easy, we can pair on it.